medical_compliance/medical_compliance/api/analyzers/heart_rate_analyzers.py

```diff
@@ -29,88 +29,50 @@
-                sent_notification = True
-
-            if mean_val > 100:
-                message = u"Jim had a high heart rate: %d between %s and %s" % \
-                    (int(mean_val), time_from, time_to)
-                description = "You can contact him and see what was wrong."
-                notifications_adapter.send_caregiver_notification(measurement_list[0].user_id, "heart", "medium", message, description)
-
-                message = u"You had a high heart rate: %d between %s and %s" % \
-                    (int(mean_val), time_from, time_to)
-                description = "Please take care."
-                notifications_adapter.send_elderly_notification(measurement_list[0].user_id, "heart", "medium", message, description)
-                sent_notification = True
-
-            if sent_notification:
-                last_timestamp = timestamps[-1]
-
-            values = [m.value]
-            timestamps = [m.timestamp]
-            window_timestamp = window_timestamp + 1200
-
-    write_last_notified_hrm_timestamp(last_timestamp)
\ No newline at end of file
+    for m in measurement_list:
+        if m.value < 60:
+            message = u"Jim's heart rate is dangerously low: only %d." % int(m.value)
+            description = "Please take action now!"
+            notifications_adapter.send_caregiver_notification(measurement_list[0].user_id, "heart", "high", message, description, m.timestamp)
+
+            message = u"Hey Jim! Your heart rate is quite low: %d." % int(m.value)
+            description = "I have contacted your caregiver."
+            notifications_adapter.send_elderly_notification(measurement_list[0].user_id, "heart", "high", message, description, m.timestamp)
+        elif m.value < 70:
+            message = u"Jim's heart rate is a bit low: only %d" % int(m.value)
+            description = "Please make sure he's all right."
+            notifications_adapter.send_caregiver_notification(measurement_list[0].user_id, "heart", "medium", message, description, m.timestamp)
+
+            message = u"Hey Jim! Your heart rate is just a bit low: %d." % int(m.value)
+            description = "How about some exercise?"
+            notifications_adapter.send_elderly_notification(measurement_list[0].user_id, "heart", "medium", message, description, m.timestamp)
+
+        if m.value > 100:
+            message = u"Jim's heart rate is dangerously high: over %d." % int(m.value)
+            description = "Please take action now!"
+            notifications_adapter.send_caregiver_notification(measurement_list[0].user_id, "heart", "high", message, description, m.timestamp)
+
+            message = u"Hey Jim! Your heart rate is quite high: %d." % int(m.value)
+            description = "I have contacted your caregiver."
+            notifications_adapter.send_elderly_notification(measurement_list[0].user_id, "heart", "high", message, description, m.timestamp)
+
+        elif m.value > 85:
```

Comment:
@rtud I didn't order them because I wanted to use the if-else flow, just as above.